templates/freeradius/sql.j2

```diff
@@ -0,0 +1,40 @@
+sql {
+  driver = "{{ freeradius_sql.driver }}"
+  dialect = "{{ freeradius_sql.dialect }}"
+{% if freeradius_sql.dialect in ['postgresql', 'mysql'] %}
+	radius_db = "{{ openwisp2_database.name }}"
```

Comment:
are you indenting this line differently in order to get the indentation right in the resulting file?
If so, we shouldn't care about that, maintainability of this role for us is more important